examples/apache-role/roles/apache-simple/handlers/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # handlers file for apache
-- name: restart apache service
+- name: restart-apache-service
```

Comment:
Interesting. I've never seen this done. What's your thinking? To take whitespace and case out of it?

Reply:
The reason for doing this is that it makes it more obvious this is not just a description.

I think it was a design-fault to use the name-key for something that is used as a reference later. So the last thing you want is name it as if it is a description. That is why I *always* recommend to use something that is clearly a variable/reference kind-of-thing. And I think it is still a good practice.

BTW Since Ansible will simply not fail when a notification handler does not exist, having someone improve the handler name (because it looks like a description) you may break a playbook without notice.